lib/routes/users.js

```diff
@@ -138,6 +138,22 @@ router.patch('/coin/stock', async (request, response) => {
   }
 });
 
+router.patch('/coin/delete', async (request, response) => {
```

Comment:
I recommend to read this:
https://stackoverflow.com/questions/15370387/restful-api-endpoint-for-adding-removing-array-elements

A more common API route would be:
`/api/users/:username/coins` with `POST` to add a new coin and `DELETE` to remove an existing coin.

Don't underestimate the importance of consistent and RESTful API routes.